spec/rubocop/result_cache_spec.rb

```diff
@@ -84,7 +131,7 @@ def abs(path)
 
   describe '#save' do
     context 'when the default internal encoding is UTF-8' do
-      let(:comments) { ["# Hello \xF0"] }
+      let(:comment_text) { '# Hello \xF0' }
```

Comment:
Oops! Should still be double quotes here, but that makes the spec fail. I'll have to fix that.